### PR TITLE
[NCP-3620] added text for new field peerSelectionAlgorithm in origin

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: zh-Hans.json [hunk 1]
```diff
@@ -1552,6 +1554,8 @@
 	"property.origin.name.on": "开启",
 	"property.origin.name.placeholder": "输入您的源站名称。",
 	"property.origin.name.remove": "删除",
+	"property.origin.name.round.ribin": "循环赛",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:1557]: Please change it to "轮询"

File: zh-Hans.json [hunk 2]
```diff
@@ -1552,6 +1554,8 @@
...
+	"property.origin.name.sorted.list": "排序列表",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:1558]: Please change it to "优选回源"

File: zh-Hans.json [hunk 3]
```diff
@@ -1540,9 +1540,11 @@
 	"property.origin.closeButton": "关闭",
 	"property.origin.dialog.name": "源站详情",
 	"property.origin.editButton": "保存",
+	"property.origin.label.peer.selection.algorithm": "对等选择算法",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:1543]: Please change it to "源站选择算法"

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]

File: zh-Hans.json (hunk 1 above)
Comment (jungho73) [reply zh-Hans.json]: modified it as suggested.

File: zh-Hans.json (hunk 2 above)
Comment (jungho73) [reply zh-Hans.json]: modified it as suggested.

File: zh-Hans.json (hunk 3 above)
Comment (jungho73) [reply zh-Hans.json]: modified it as suggested.

**Review by Jianghai-Quantil — APPROVED** [review]

**Comment by nibinqtl** [comment]
Hold on. There are still problems.

File: ar_SA.json [hunk 4]
```diff
@@ -1552,6 +1554,8 @@
 	"property.origin.name.on": "على",
 	"property.origin.name.placeholder": "Enter a name for your origin.",
 	"property.origin.name.remove": "إزالة",
+	"property.origin.name.round.ribin": "جولة روبن",
```
Comment (nibinqtl) [inline ar_SA.json:1557]: Please change ribin to robin

File: zh-Hans.json [hunk 5]
```diff
@@ -1540,9 +1540,11 @@
 	"property.origin.closeButton": "关闭",
 	"property.origin.dialog.name": "源站详情",
 	"property.origin.editButton": "保存",
+	"property.origin.label.peer.selection.algorithm": "源站选择算法",
```
Comment (nibinqtl) [inline zh-Hans.json:1543]: Please change the value to "IP选择算法”， to be consistent to the peer failure timeout translation.

**Review by nibinqtl — CHANGES_REQUESTED** [review]

File: zh-Hans.json (hunk 5 above)
Comment (jungho73) [reply zh-Hans.json]: fixed the name and changed some text for consistency.

File: ar_SA.json [hunk 6]
```diff
@@ -1552,6 +1554,8 @@
 	"property.origin.name.on": "على",
 	"property.origin.name.placeholder": "Enter a name for your origin.",
```
Comment (burtonquantil) [inline ar_SA.json:1555]: Does property.origin.name.placeholder ever get shown to the user?

Comment (jungho73) [reply ar_SA.json:1555]: Yes, it shows in origin details
image

Comment (burtonquantil) [reply ar_SA.json:1555]: If it is shown, why isn't it also translated ?

Comment (jungho73) [reply ar_SA.json:1555]: As we already know, if you look at the files, most of them are in English and we started to translate them at some point. Currently, we translate into all supported languages, but many previous parts are still supported in English.

**Comment by jungho73** [comment]
Is there anything else that needs to be modified?

**Review by nibinqtl — APPROVED** [review]
looks good enough now.

**Pull request merged**